user-creation/src/test/java/no/unit/nva/useraccessservice/usercreation/person/cristin/exceptions/IdentityServiceExceptionsTest.java

```diff
@@ -0,0 +1,276 @@
+package no.unit.nva.useraccessservice.usercreation.person.cristin.exceptions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.useraccessservice.usercreation.person.IdentityServiceErrorCodes;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import org.junit.jupiter.api.Test;
+
+class IdentityServiceExceptionsTest {
```

Comment:
These tests are a bit weird. They test the `Problem.class`, rather than our use of this class. I don't see the point.